### PR TITLE
more complex fixOrientation

### PR DESCRIPTION
**Pull request by martenb**

### CODE REVIEW

File: src/Modifiers/BaseModifiers.php [hunk 1]
```diff
@@ -53,18 +53,18 @@ public function load(IModifiers $modifiers): void {
...
-						$image->rotate(-90, 0);
-						break;
-
-					case 8:
-						$image->rotate(90, 0);
-						break;
+				if (in_array($exif['Orientation'], [3, 4])) {
+					$image->rotate(180, 0);
+
+				} elseif (in_array($exif['Orientation'], [5, 6])) {
```
Comment (MartkCz) [inline src/Modifiers/BaseModifiers.php]: elseif => else if

File: src/Modifiers/BaseModifiers.php [hunk 2]
```diff
@@ -53,18 +53,18 @@ public function load(IModifiers $modifiers): void {
...
+					$image->rotate(-90, 0);
+
+				} elseif (in_array($exif['Orientation'], [7, 8])) {
```
Comment (MartkCz) [inline src/Modifiers/BaseModifiers.php]: elseif => else if

**Review by MartkCz — COMMENTED** [review]
after cs fix, I'll merge it

**Comment by martenb** [comment]
cs fixed, do you have some command to check cs? maybe you should add it to composer.json...

**Comment by MartkCz** [comment]
Not now, cs is going to change, thanks for pr

**Pull request merged**